projects/kit/directives/highlight/highlight.directive.ts

```diff
@@ -7,6 +7,8 @@ import {svgNodeFilter} from '@taiga-ui/cdk/constants';
 import {tuiInjectElement} from '@taiga-ui/cdk/utils/dom';
 import {tuiPx} from '@taiga-ui/cdk/utils/miscellaneous';
 
+import {TUI_HIGHLIGHT_OPTIONS} from './highlight.options';
```

Comment:
My goal of making that `tuiCreateOptions` to make options generation as short as possible. Perhaps we do not need a whole file for it and just put it right here?

Reply:
Agree [fede712](https://github.com/taiga-family/taiga-ui/pull/8812/commits/fede712345c87ee2ef7f12a2c37275bc66116d55)